### PR TITLE
Update taint-and-toleration.md

### PR DESCRIPTION
**Pull request by ravann**

### CODE REVIEW

File: content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ kubectl taint nodes node1 key1=value1:NoSchedule
 places a taint on node `node1`. The taint has key `key1`, value `value1`, and taint effect `NoSchedule`.
 This means that no pod will be able to schedule onto `node1` unless it has a matching toleration.
 
-To remove the taint added by the command above, you can run:
+To remove the taint added by the command above, you can run (notice the - at the end of the taint):
 ```shell
 kubectl taint nodes node1 key1=value1:NoSchedule-
```
Comment (RA489) [inline content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md:43]: I think this command is self sufficient to describe it. Do we really need to highlight `-` explicitly?

File: content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ kubectl taint nodes node1 key1=value1:NoSchedule
 places a taint on node `node1`. The taint has key `key1`, value `value1`, and taint effect `NoSchedule`.
 This means that no pod will be able to schedule onto `node1` unless it has a matching toleration.
 
-To remove the taint added by the command above, you can run:
+To remove the taint added by the command above, you can run (notice the - at the end of the taint):
```
Comment (RA489) [inline content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md:41]: I think this extra line is not required. Current text is self sufficient to describe it.

File: content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md (hunk 1 above)
Comment (ravann) [reply content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md:43]: It wasn't obvious to me how to remove the taint.  Had to google to understand this.  Thought additional comment will help people like me to understand it better.

**Comment by tengqm** [comment]
CLA not signed.
Change unnecessary.
/close

**Pull request closed without merging**